src/detections/configs.rs

```diff
@@ -619,6 +658,10 @@ pub struct CsvOutputOption {
     #[arg(short, short = 'J', long = "JSON-input")]
     pub json_input: bool,
 
+    /// Scan JSON-formatted logs instead of .evtx
```

Comment:
```suggestion
    /// Add log enhancement for IP address (ASN, City and Country: Required MaxMind GeoIP databases)
```

Reply:
I would appreciate it if you could fix the typo :)

Reply:
thanks for your review.
i merged your suggestion.